src/layouts/EditResourcePage.jsx

```diff
@@ -97,11 +97,13 @@ export default class EditResourcePage extends Component {
   render() {
     const { match } = this.props;
     const { siteName, fileName, resourceName } = match.params;
+    const { title, date, type } = prettifyResourceFileName(fileName);
```

Comment:
This was a problem that the original code author introduced, but could we rename this to something that reflects that we're extracting resource metadata, as opposed to just prettifying the file name?

Reply:
Resolved in https://github.com/isomerpages/isomercms-frontend/pull/167/commits/1f3f6fc20f6288af47b6c3c5357609a2869b0404